src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/TokenExpandable.vue

```diff
@@ -196,6 +251,21 @@ export default Vue.extend({
 
       return ''; // if TokenUIStatus is Active or Available or unknown return empty string
     },
+
+    tokenIcon(): string {
+      const status: TokenUIStatus = getTokenUIStatus(this.token.status);
+
+      if (status === TokenUIStatus.Inactive) {
+        return 'icon-Cancel';
```

Comment:
Perhaps add an enum for TokenStatusIcons

Reply:
I decided not to because these are used only once